cli.js

```diff
@@ -19,16 +20,30 @@ program
   .description(
     "Translate a text and put the result on the files in the output directory",
   )
-  .option("-t, --text <string>", "Text to translate")
-  .option("-f, --from <string>", "Source language of the string")
-  .option("-n, --name <string>", "Name of the translation")
+  .option(
+    "-t, --text <string>",
+    "The word or sentence that you want to translate.",
+  )
+  .option(
+    "-f, --from <string>",
+    "The language of the text that you wrote on the --text option.",
+  )
+  .option(
+    "-n, --name <string>",
+    "The name of the property that you want your text has on the output files.",
+  )
   .option(
     "-e, --engine <string>",
-    "[OPTIONAL]. Engine to use for the translation. In case you don't define it will be use by default all the translation engines that are free and doesn't requires API Key.",
+    `[OPTIONAL]. The engine that you want to use to translate the text. Available options: ${validEngines.join(", ")}
+    If you specify a engine on the command, it will put it on the first position of the array of engines.
```

Comment:
In the example have some extra space at the beginning of the line, maybe check alternatives for better readability

Reply:
will take a look. Thanks!

Reply:
not sure if this is better?

idea 1:
![image](https://github.com/heliomar-pena/i18n-populator/assets/66505715/b4434bdc-8b9b-461c-b7ca-8f20faef54aa)

idea 2:
![image](https://github.com/heliomar-pena/i18n-populator/assets/66505715/c2a5134f-2f9d-4ff5-8c3f-ca7ed9915540)

or if you have any other idea you wanted try?

Reply:
> not sure if this is better?
> 
> idea 1: ![image](https://private-user-images.githubusercontent.com/66505715/347156964-b4434bdc-8b9b-461c-b7ca-8f20faef54aa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2NDQxNDAsIm5iZiI6MTcyMDY0Mzg0MCwicGF0aCI6Ii82NjUwNTcxNS8zNDcxNTY5NjQtYjQ0MzRiZGMtOGI5Yi00NjFjLWI3Y2EtOGYyMGZhZWY1NGFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzEwVDIwMzcyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJiZmZkNzdjMDFmYTMwMTQzZDc5NWE3Y2U1ZjZjMjI3NWUxMTNlNGYxMzYzZDNiZDU1MWFkYzlkYWNiNmU0MTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-tcdMV-a73x3SjlsdyCUh6qMYsLkUSDFE6DHeyey7Ms)
> 
> idea 2: ![image](https://private-user-images.githubusercontent.com/66505715/347157097-c2a5134f-2f9d-4ff5-8c3f-ca7ed9915540.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2NDQxNDAsIm5iZiI6MTcyMDY0Mzg0MCwicGF0aCI6Ii82NjUwNTcxNS8zNDcxNTcwOTctYzJhNTEzNGYtMmY5ZC00ZmY1LThjM2YtY2E3ZWQ5OTE1NTQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzEwVDIwMzcyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIyNWJmN2Q5MTliOWM2NTZkZDQxYzNhNDViYmFlNGYxNTZjYThjYzM1NWQ2ODMzNTUzZWM5YzY0Mzg2YjFkOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.THNFq1iGBBuBbF4VEYqWET3NVQMSMc8Fv0G4cUOPJO0)
> 
> or if you have any other idea you wanted try?

@YoelFerreyra can you help us to decide which looks better and is easier to read?

Reply:
I like number 1